xlive/H2MOD/Modules/Shell/Config.cpp

```diff
@@ -724,12 +724,12 @@ void ReadH2Config() {
 					case 0:
 						H2Config_experimental_fps = _rendering_mode_none;
 						break;
-					case 1:
+					/*case 1:
 						H2Config_experimental_fps = _rendering_mode_old;
 						break;
 					case 2:
 						H2Config_experimental_fps = _rendering_mode_new;
-						break;
+						break;*/
 					case 3:
 						H2Config_experimental_fps = _rendering_mode_original_game_frame_limit;
 				}
```

Comment:
Can we clean this up so we don't have dead code lying around, so we just have options 0 and 1 for none and _rendering_mode_original_game_frame_limit

Reply:
Want the functionality to be easily re-addable?